opm/autodiff/StandardWell_impl.hpp

```diff
@@ -395,7 +395,8 @@ namespace Opm
     computePerfRate(const IntensiveQuantities& intQuants,
                     const std::vector<EvalWell>& mob_perfcells_dense,
                     const double Tw, const EvalWell& bhp, const double& cdp,
-                    const bool& allow_cf, std::vector<EvalWell>& cq_s) const
+                    const bool& allow_cf, std::vector<EvalWell>& cq_s,
+                    double& perf_dis_gas_rate, double& perf_vap_oil_rate) const
```

Comment:
If possible, these should also be of type `EvalWell&` rather than `double`.

Reply:
These values are supposed to enter `WellState` for output. We are not using them in anyway in our simulation. I will suggest we only keep the value here.  Please suggest.

Reply:
Other comments were address except this one.